code/game/objects/items/passport.dm

```diff
@@ -31,9 +31,9 @@
 	to_chat(user, info || SPAN_WARNING("[src] is completely blank!"))
 
 /obj/item/passport/scg
```

Comment:
Search and replace this type with `/obj/item/passport/goc`

Reply:
fixed